### DISCUSSION TITLE
First time user: Could not run example using VCS simulator

### DISCUSSION THREAD

**Discussion by ntomj** [cocotb/cocotb#3958] [Q&A]
Hello,

I want to run examples provided https://docs.cocotb.org/en/stable/quickstart.html.
VCS is the simulator I am using V-2023.12-SP1 and Python is 3.11.

`pip install cocotb
export PATH=/home/bob/.local/lib/python3.11/site-packages/cocotb:$PATH
cocotb-config
make SIM=vcs
`

**Round 1:**
The errors

<pre>Error-[OBSOLETE_OPTION_PLUSACC] Obsolete option &apos;+acc&apos; used
  The specified option &apos;+acc+1&apos; is no longer supported.
  Please use &apos;-debug_access+r+w-memcbk -debug_region+cell&apos; instead.

     -.--ns INFO     gpi                                ..mbed/gpi_embed.cpp:76   in set_program_name_in_venv        Did not detect Python virtual environment. Using system-wide Python interpreter
     -.--ns INFO     gpi                                ../gpi/GpiCommon.cpp:101  in gpi_print_registered_impl       VPI registered
</pre>

**Round 2:**

I edited Makefile.vcs to replace it with -debug_access+...  and added +vcs+lic+wait as we were out of simulator licenses at that time.

The error is attached.
![cococtb_err](https://github.com/cocotb/cocotb/assets/91078656/b35a923d-35e8-4ca3-b273-dfc1754e9804)

Can someone please guide me how do I go about debugging the issue? 

Thanks.

**Comment by ktbarrett** [comment]
This was fixed on master yesterday https://github.com/cocotb/cocotb/pull/3951

**Reply by ntomj** [reply]
Thanks for the quick response. Do you know if the release is available? 
pip install --upgrade cocotb still has 1.8.1 release.

**Reply by ktbarrett** [reply]
1.9 won't be released for at least another week, but probably even more.

**Reply by ntomj** [reply]
I am a novice here but I think my problem is more than outdated VCS arguments. I have fixed them locally but I still have trouble with cocotb_tools. Can you please help?

**Reply by ktbarrett** [reply]
https://app.gitter.im/#/room/#cocotb_Lobby:gitter.im might be the better place for this, but sure, I'm willing to help.

**Reply by ntomj** [reply]
Thanks, I have posted it in there.

**Reply by benjacam** [reply]
I think your problem was the same as this issue: https://github.com/cocotb/cocotb/issues/3809
